packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -678,12 +669,12 @@ export function ReorderExample(props) {
             let key;
```

Comment:
There is a slight difference between those two onDrop blocks. One has:
```
        if (!keys.includes(e.target.key)) {
          onMove(keys, e.target);
        }
```
which is to prevent the user from performing a drag operation that places a folder inside itself. The reorder example doesn't have this logic in its onDrop because it doesn't have to account for this issue. The other stories can be refactored, but out of scope of this PR